spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -1309,8 +1309,10 @@ def function(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
       # absolute Exclude paths will point into this example's work directory.
       RuboCop::ConfigLoader.default_configuration = nil
 
+      $stdout = StringIO.new
       expect(cli.run(['--auto-gen-config', '--auto-gen-only-exclude',
                       '--exclude-limit', '1'])).to eq(0)
+      expect($stdout.string.include?('run Layout/LineLength cop (skipped')).to be(true)
```

Comment:
I think you should test the whole message about why `Layout/LineLength` is skipped. Don't cut it after `(skipped` because that could potentially be a different message.

Reply:
Good call. Done!